extensions/interactions/base_test.py

```diff
@@ -256,6 +256,7 @@ def test_default_interactions_are_valid(self):
             #    * A python file called {InteractionName}.py.
             #    * An __init__.py file used to import the Python file.
             #    * An html file called {InteractionName}.html.
```

Comment:
Should we delete this one?

Actually I'm a bit confused by the changes here. From the stuff up here it looks like we've added a new TS file. But from the stuff below we seem to have replaced the html file validation with the TS file validation. These seem a bit inconsistent?

Also should we be naming the {{InteractionName}}.ts file something like {{InteractionName}}.requires.ts instead, to be clearer about what it is?

Lastly, I think we currently keep all our ts files for interactions in static/. Not sure if we should follow that convention with this one as well?

Reply:
Add this in the comment please, so that devs know that the HTML files are scheduled for deletion. Link to the issue (like you've done in other comments).

Reply:
Done.

Reply:
I have also added myself as an code owner for interactions until we fully migrate them to webpack.